test/test_helper.rb

```diff
@@ -9,38 +9,26 @@
 
 module MiniTest
   class Test
-    def raw_data_fixture(domain)
-      File.read("test/data/articles/#{domain.tr('.', '_')}_raw")
-    end
+    def setup
+      super
 
-    def transformation_fixture(domain)
-      YAML.load_file("test/data/articles/#{domain.tr('.', '_')}_transformed.yml")
-    end
+      NewsScraper.reset_configuration
```

Comment:
Cause NewsScraper configuration is a class global whatever variable and not resetting will make this leak into new tests and then "kaboom"